src/utils/typography.js

```diff
@@ -7,6 +7,7 @@ import Wordpress2016 from 'typography-theme-wordpress-2016';
 Wordpress2016.overrideThemeStyles = () => ({
   a: {
     color: 'var(--textLink)',
+    'padding-bottom': '2px',
```

Comment:
```suggestion
    paddingBottom: '2px',
```

Or maybe even

```suggestion
    paddingBottom: 2,
```

Reply:
I really like the first approach! But I would personally prefer using px instead of nothing to make it easier to understand and the units are used in the other rules as well